### PR TITLE
add helm charts for all jobs

### PR DESCRIPTION
**Pull request by pcarney8**
#### What is this PR About?
Describe the contents of the PR

#### How do we test this?
Provide commands/steps to test this PR.

cc: @redhat-cop/casl

### CODE REVIEW

File: charts/cluster-descheduler/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+data:
+  policy.yaml: "apiVersion: descheduler/v1alpha1\nkind: DeschedulerPolicy\nstrategies:\n
```
Comment (jkupferer) [inline charts/cluster-descheduler/templates/configmap.yaml]: Can we use a YAML block quote here?
```
data:
  policy.yaml: |
    apiVersion: descheduler/v1alpha1
    ...
```

**Review by jkupferer — CHANGES_REQUESTED** [review]

File: charts/cluster-descheduler/templates/configmap.yaml (hunk 1 above)
Comment (pcarney8) [reply charts/cluster-descheduler/templates/configmap.yaml]: updated.

File: charts/cluster-descheduler/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+description: Descheduler to remove pods violating usage thresholds, affinity violations
+  and dupliacate pod placements
```
Comment (mcanoy) [inline charts/cluster-descheduler/Chart.yaml:2]: duplicate typo

**Review by oybed — APPROVED** [review]
LGTM - any nitpick will be addressed in follow-up

**Pull request merged**